.github/workflows/draft-release-notes-workflow.yml

```diff
@@ -1,14 +1,18 @@
 name: Release Drafter
 
 on:
+  pull_request:
+    types: [opened, reopened]
   push:
     branches:
-      - main
+      - 'main'
+      - '[1-9]+.[0-9x]+'
+      - '[1-9]+.[0-9x]+.[0-9x]+'
```

Comment:
what is `'[1-9]+.[0-9x]+.[0-9x]+'` for?

Reply:
It is regex-like branch filter, it matches branch names like 3.3, 1.x, 2.3.x.

Reply:
i meant to ask is this needed? since i think we only have x.y branch, not x.y.z

Reply:
Fixed in 63c51377.